docs/source/reference_aggregation.rst

```diff
@@ -0,0 +1,191 @@
+Other Methods of Aggregations
+=============================
+
+As we have already seen in this `page <https://fsspec.github.io/kerchunk/test_example.html#multi-file-jsons>`_,
+that the main purpose of ``kerchunk`` it to generate references, to view whole archive of files like
+GRIB2, NetCDF etc. allowing us for *in-situ* access to the data. In this part of the documentation,
+we will see some other efficient ways of combining references.
+
+GRIB Aggregations
+-----------------
+
+This new method for reference aggregation, developed by **Camus Energy**, is based on GRIB2 files. Utilizing
```

Comment:
I won't be "new" for long, so drop this.

I would put the restrictions first:
- must have .idx files
- specialised for time-series data, each file having identical message structure

@emfdavid : an opinion on whether Camus wants to be referenced here and how that would look.

Reply:
@emfdavid  Should I mention `Camus Energy` like @martindurant  asked here?

Reply:
I checked in with the team and it would be great if you could link https://www.camus.energy/ and if there is a place for attribution you can include my github handle @emfdavid, otherwise the contributors list is fine.